prometheus-extension-ballerina/metric_reporter.bal

```diff
@@ -110,7 +105,7 @@ isolated function startReporter(string host, int port) returns error? {
 # + metricType - Type of Metric.
 # + return - Formatted metric information.
 isolated function generateMetricInfo(string name, string metricType) returns string {
-    return "# TYPE " + name + " " + metricType + "\n";
+    return string `# TYPE ${name} ${metricType}`;
```

Comment:
Isn't it better to return a string array from these also and simply concatenate the array ? Then finally we can join

Reply:
If we do so the places that read `metricInfo` need to use `'join` with `' '`.
ie:
https://github.com/ballerina-platform/module-ballerinax-prometheus/blob/483c9d79f593585b726710289fa3686aaf7890e8/prometheus-extension-ballerina/metric_reporter.bal#L71

But in some places, we use `\n` to join.
ie:
https://github.com/ballerina-platform/module-ballerinax-prometheus/blob/483c9d79f593585b726710289fa3686aaf7890e8/prometheus-extension-ballerina/metric_reporter.bal#L94

So I feel that would make things complex to a reader if we send string arrays from functions like `generateMetricInfo`.

On the performance wise also there is no much gain doing so isnt'it?